src/progressive-profiling/CustomFormFields.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import { FormFieldRenderer } from '../field-renderer';
```

Comment:
```suggestion
import React from 'react';
import PropTypes from 'prop-types';
import { FormFieldRenderer } from '../field-renderer';
```

Does this break any codestyle? 
Looks better for me.

Reply:
linter says: `There should be at least one empty line between import groupseslint import/order`

Reply:
This is how the linter is configured for this MFE